akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerMTSpec.scala

```diff
@@ -14,16 +14,16 @@ import akka.testkit._
 class CircuitBreakerMTSpec extends AkkaSpec {
   implicit val ec: ExecutionContextExecutor = system.dispatcher
   "A circuit breaker being called by many threads" must {
-    val callTimeout = 2.second.dilated
+    val callTimeout = 2.seconds.dilated
     val resetTimeout = 3.seconds.dilated
     val maxFailures = 5
-    val breaker = new CircuitBreaker(system.scheduler, maxFailures, callTimeout, resetTimeout)
+    def newBreaker = new CircuitBreaker(system.scheduler, maxFailures, callTimeout, resetTimeout)
     val numberOfTestCalls = 100
 
-    def openBreaker(): Unit = {
+    def openBreaker(breaker: CircuitBreaker): Unit = {
       // returns true if the breaker is open
       def failingCall(): Boolean =
-        Await.result(breaker.withCircuitBreaker(Future(throw new RuntimeException("FAIL"))).recover {
+        Await.result(breaker.withCircuitBreaker(Future.failed(new RuntimeException("FAIL"))).recover {
```

Comment:
I don't think we need to schedule this call, so just returning a failed future right away.

Reply:
No optimization in the breaker on already failed vs failing a bit later future? (Scheduling it on ec doesn't really make for a great reproducible case for that either though)

Reply:
What this method is doing is to send as many failed futures as needed to open the CB. We want to open it as soon as possible. So scheduling doesn't make sense for us here.